server/core/store.go

```diff
@@ -28,9 +28,11 @@ import (
 
 const (
 	// Interval to save store meta (including heartbeat ts) to etcd.
-	storePersistInterval = 5 * time.Minute
-	mb                   = 1 << 20 // megabyte
-	gb                   = 1 << 30
+	storePersistInterval   = 5 * time.Minute
+	mb                     = 1 << 20 // megabyte
+	gb                     = 1 << 30
+	initialMaxRegionCounts = 30          // exclude storage Threshold Filter when region less than 30
+	minimumSpace           = 10737418240 // 10GB
```

Comment:
The naming does not express the true meaning. It is best to add comments to these two parameters.
Use `1 << 30` to make it more readable.